udata/auth/__init__.py

```diff
@@ -40,20 +40,13 @@ def __init__(self, *needs):
 
 def init_app(app):
     from .forms import ExtendedRegisterForm, ExtendedLoginForm, ExtendedResetPasswordForm
-    from .tasks import sendmail_proxy
-    from .views import create_security_blueprint
-    from .password_validation import password_validator
+    from .mails import UdataMailUtil
+    from .password_validation import UdataPasswordUtil
     from udata.models import datastore
-    state = security.init_app(app, datastore,
-                              register_blueprint=False,
-                              login_form=ExtendedLoginForm,
-                              confirm_register_form=ExtendedRegisterForm,
-                              register_form=ExtendedRegisterForm,
-                              reset_password_form=ExtendedResetPasswordForm,
-                              send_mail=sendmail_proxy
-                            )
-    state.password_validator(password_validator)
-
-    security_bp = create_security_blueprint(state, 'security_blueprint')
-
-    app.register_blueprint(security_bp)
```

Comment:
- Here I deleted our custom blueprint to try to use built in Flask security blueprint and translations. I passed Udata's i18n domain we declare in the app initialisation through the setting `SECURITY_I18N_DOMAIN`.
This seems to work but new translations need to be filled through crowdin.
The argument `register_blueprint=False` was used to tell flask security to not register its built in blueprint.

- `state.password_validator` method does not exist anymore. It now takes a Python object through the argument `password_util_cls` during the init. I translated our `password_validator` fund to a `UdataPasswordUtil` class

- Same for `send_mail`. It now takes a Python object through the argument `mail_util_cls` during the init. I translated our `sendmail_proxy` fund to a `UdataMailUtil` class